migrations/versions/9155f0d3d028_.py

```diff
@@ -0,0 +1,64 @@
+"""set TLS v1.2 for existing ldap resolvers while introducing the TLS v1.3 for new ones
+
+Revision ID: 9155f0d3d028
+Revises: d5870fd2f2a4
+Create Date: 2021-03-23 14:25:48.425762
+privacyIDEA Version: 3.6
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '9155f0d3d028'
+down_revision = 'd5870fd2f2a4'
+
+from alembic import op
+from sqlalchemy import orm
+from privacyidea.models import Resolver, ResolverConfig
```

Comment:
You need to copy the Class definition of the DB ``Resolver`` table and the ``ResolverConfig`` table here!

Otherwise the update script will fail if in later version the table ``Resolver`` or ``ResolverConfig`` would be enhanced.
I can explain this in detail.